src/components/TextInputFocusable/index.js

```diff
@@ -8,7 +8,10 @@ const propTypes = {
     maxLines: PropTypes.number,
 
     // The default value of the comment box
-    defaultValue: PropTypes.string.isRequired,
+    defaultValue: PropTypes.string,
+
+    // The value of the comment box
+    value: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
```

Comment:
Can we pick one type or the other? Why should we accept both `string` and `number`?